usage_windows.go

```diff
@@ -41,11 +59,10 @@ func customTargets() string {
 
 func platformCommandLineParameters() string {
 	return `
-    --nssm NSSM-EXE                         The full path to nssm.exe to use for installing
-                                            the service.
-                                            [default: C:\nssm-2.24\win64\nssm.exe]
     --service-name SERVICE-NAME             The name that the Windows service should be
-                                            installed under. [default: Generic Worker]`
+                                            installed under. [default: Generic Worker]
+    --working-directory DIRECTORY           The working directory the Generic Worker
+                                            service will use. [default: C:\Windows\system32]`
```

Comment:
Since the worker writes files to the working directory, can we change the default to e.g. `C:\generic-worker`? Perhaps the correct drive letter can be determined from the system, in case the operating system isn't installed on C:.

Reply:
What would the behavior be if that directory doesn't exist?

Reply:
I think I'm going to change the behavior here.

If no `--working-directory` is specified, we default to the parent directory of the generic-worker executable. If one is specified, we use it.